### ISSUE TITLE
Possibility of sending fake 'typing...' or 'recording...'

### ISSUE THREAD

**Issue by MirzaHasnat** [tomer8007/whatsapp-web-incognito#169]
is there any possibility we can add a feature whcih will send a fake status to my speicific friend that im typing or recording?

**Comment by tomer8007** [comment]
I think this is a weird feature. Why would you want to send a fake "typing" when you can just type?

**Comment by MirzaHasnat** [comment]
Yeah its little wired but for example im on another tab and i toggle the typing feature till i turn it off


Just for pranking.

**Comment by tomer8007** [comment]
I don't think it's worth it.
